src/main/java/book/store/intro/dto/book/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package book.store.intro.dto.book;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
+    @NotNull
+    @Min(0)
     private BigDecimal price;
```

Comment:
Please make validations more useful.
Check where reasonable to apply `@NotEmpty`, `@NotBlank`, `@Length`, `@Isbn` and other annotations